Mathlib/GroupTheory/Index.lean

```diff
@@ -559,6 +561,19 @@ end FiniteIndex
 
 end Subgroup
 
+namespace MulAction
+
+theorem index_stabilizer (G : Type*) {X : Type*} [Group G] [MulAction G X] (x : X) :
+    (stabilizer G x).index = (orbit G x).ncard :=
+  (Nat.card_congr (MulAction.orbitEquivQuotientStabilizer G x)).symm.trans
+    (Set.Nat.card_coe_set_eq (orbit G x))
+
+theorem index_stabilizer' (G : Type*) {X : Type*} [Group G] [MulAction G X] [IsPretransitive G X]
```

Comment:
There's recently been some discussion against primes in names - can you try thinking of another way to distinguish these two?

Reply:
good point, I renamed it `index_stabilizer_of_transitive`